SpatialGDK/Source/SpatialGDK/Public/Interop/OwnershipCompletenessHandler.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "Misc/Optional.h"
+
+#include "SpatialCommonTypes.h"
+#include "SpatialView/SubView.h"
+
+class USpatialNetDriver;
+
+namespace SpatialGDK
+{
+class ViewCoordinator;
+
+class FOwnershipCompletenessHandler
+{
+public:
+	FOwnershipCompletenessHandler(bool bInIsServer)
+		: bIsServer(bInIsServer)
```

Comment:
nit - no programming need too, but personally, if I use initialisers to set one member variable I prefer to include them all.

Reply:
Interesting - can't say I've considered doing this.